os/src/ProcessOps.scala

```diff
@@ -68,7 +99,34 @@ object spawn {
       stdout = stdout,
       stderr = stderr,
       mergeErrIntoOut = mergeErrIntoOut,
-      propagateEnv = propagateEnv
+      propagateEnv = propagateEnv,
+      shutdownGracePeriod = shutdownGracePeriod,
+      destroyOnExit = destroyOnExit
+    )
+  }
+  def apply(
```

Comment:
Is this only provided for bin-compat? Shouldn't we deprecate it therefore?

Reply:
I've had issues with `@deprecated` things being accidentally resolved here before, but will add a comment like the other forwarders